packages/swap/src/quoting/quotes.ts

```diff
@@ -83,3 +132,126 @@ export const buildQuoteRequest = (query: QuoteQueryParams): QuoteRequest => {
     deposit_amount: amount,
   };
 };
+
+export const calculateIncludedFees = (
+  request: QuoteRequest,
+  quote: MarketMakerResponse,
```

Comment:
The `MarketMakerResponse` can technically also be a `BrokerQuoteResponse` too right? are we just calling them both `MarketMakerResponse` ?

Reply:
yes unfortunately we have a single type only for both of them at the moment. i will add a `BrokerQuoteResponse` type to make this explicit